tests/integration/components/hyper-table-v2-test.ts

```diff
@@ -224,6 +371,54 @@ module('Integration | Component | hyper-table-v2', function (hooks) {
         .isNotChecked();
       assert.dom('.selected-count').hasText('0');
     });
+
+    test('clicking on all row selection checkbox toggles the select all', async function (assert: Assert) {
```

Comment:
all row selection checkbox

please use the same denomation for all the tests (Header checkbox, select-all checkbox, multiline checkbox, etc) 🙏